RAMP/unpacker.py

```diff
@@ -59,8 +62,16 @@ def _validate_zip_file(zip_file, max_bundle_size_mb):
     exceed a certain threshold.
     """
     infolist = zip_file.infolist()
-    # We're expecting exactly two files.
-    if len(infolist) != 2:
+    # We're expecting to have .so file, module.json and deps dir.
+    files_counter = 0
+
+    for file in infolist:
+        if file.filename.endswith(".so") or file.filename == "module.json":
+            files_counter += 1
+        elif not file.filename.startswith("/deps"):
+            files_counter -= 1
+
+    if files_counter != 2:
```

Comment:
I don't understand, this looks like a hack...
files counter can obviously be > 2

Reply:
I want to make sure there exactly two files (module.so and module.json)+ deps dir which contains as many files as needed.